main.c

```diff
@@ -18,17 +18,34 @@ int main(int argc, char* argv[])
 		printf("Usage: %s n\n", argv[0]);
 		return 1;
 	}
+	if(argv[1]<0){
+		printf("Negative arguments are not allowed!");
+		return 1;
+	}
 	
 	int n = atoi(argv[1]);
 	int f = 0;
 	
 	printf("%u", n);
 	
 	greet();
+        int row=0,col=0;
 	while(!f)
 	{
 		clear();
-		show(n, &n);
+                printf("\nPlayer 1: Enter Postion:\nRow Number: ");
+                row=getchar();
+                printf("\nColumn Number: ");
+                col=getchar();
+		ARR[row][col]='X';
```

Comment:
Same space thing. + The name of the array needs to be changed. 
There might be a new issue targeting just on the coding structure